docs/generators.md

```diff
@@ -145,6 +148,16 @@ type, making their usage much nicer. These are
 
 > `range()` for floating point numbers has been introduced in Catch2 2.11.0
 
+### Random generators
+```cpp
+#include <catch2/generators/catch_generators_random.hpp>
+```
+
```

Comment:
But I don't like it for lists, and I don't like adding new sections just to have the include paths in them.

Reply:
I think there is enough extra information to be mentioned about the random generators (e.g. their repeatability guarantees) that this subheading could be filled out to be meaningful.

However, I don't see this also being true for the `range` generator, so we will need a different format for inline markings of things in lists.